src/amino/arkeo/index.ts

```diff
@@ -0,0 +1,359 @@
+import { AminoMsg } from '@cosmjs/amino';
+import { fromBech32, toBech32 } from '@cosmjs/encoding';
+import { AminoConverters } from '@cosmjs/stargate';
+
+import * as codecs from '../../proto';
+import * as cosmos from '../../proto/generated/cosmos/base/v1beta1/coin'
+import { ProviderStatus, ContractType, ContractAuthorization } from '../../proto/generated/arkeo/arkeo.arkeo/types/arkeo/arkeo/keeper'
+import { Chain } from '../../proto/generated/arkeo/arkeo.claim/types/arkeo/claim/claim_record'
+
+
+export interface AminoMsgBondProvider extends AminoMsg {
+    readonly type: 'arkeo/MsgBondProvider',
```

Comment:
these types don't match the type specified in the amino codec registration:
https://github.com/arkeonetwork/arkeo/blob/fe74a092d5d8335d8dbe9c439cff2e7af6443d4f/x/arkeo/types/codec.go